### DISCUSSION TITLE
Welcome to epubjs-react-native Discussions!

### DISCUSSION THREAD

**Discussion by victorsoares96** [victorsoares96/epubjs-react-native#170] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gustafstr** [comment]
Hey Victor, Love your build. Quick question, currently it seems like the reader only supports https links to a book, any ideas how to upload files in the app and then read them through the reader?
Keep up the good work!

**Reply by victorsoares96** [reply]
Hi @Gustafstr, this is now possible! Check the [examples](https://github.com/victorsoares96/epubjs-react-native/blob/master/example-expo/examples/Formats/index.tsx) provided in the repository

**Reply by Gustafstr** [reply]
Cool, thanks for that!

**Comment by meltar95** [comment]
How can we open DRM-protected epub files with a license key?

**Comment by ayushguptax** [comment]
Hi @victorsoares96,

Awesome work with the library, how can we use `initialBookmark` reader params ? I am trying to pass a copy of `bookmarks` array but it gets stuck in re-render and I can see bookmarks being updated again and again.

Can you show some light on using the `initialBookmark` reader params ?
Any help would be grateful.
